geopandas/tests/test_geom_methods.py

```diff
@@ -326,6 +327,21 @@ def test_is_simple(self):
         expected = Series(np.array([True] * len(self.g1)), self.g1.index)
         self._test_unary_real('is_simple', expected, self.g1)
 
+    def test_xy_points(self):
+        expected_x = [-73.9847, -74.0446]
+        expected_y = [40.7484, 40.6893]
+
+        assert_array_equal(expected_x, self.landmarks.x)
+        assert_array_equal(expected_y, self.landmarks.y)
+
+    def test_x_polygons(self):
+        with pytest.raises(ValueError):
+            x = self.gdf1.x
+
+    def test_y_polygons(self):
```

Comment:
you can put those two together in the first test for x and y attributes (just separate with a comment)